app/ts/simulation/services/EthereumClientService.ts

```diff
@@ -198,7 +198,16 @@ export class EthereumClientService {
 		const httpsRpc = this.requestHandler.getRpcEntry().httpsRpc
 		if (httpsRpc === 'https://rpc.dark.florist/winedancemuffinborrow' || httpsRpc === 'https://rpc.dark.florist/birdchalkrenewtip') {
 			//TODO: Remove this when we get rid of our old multicall
-			return this.executionSpec383MultiCallOnlyTransactionsAndSignatures(transactions, spoofedSignatures, blockNumber, extraAccountOverrides)
+			
+			const transactionsWithRemoveZeroPricedOnes = transactions.map((transaction) => {
+				if (transaction.type !== '1559') return transaction
+				const { maxFeePerGas, ...transactionWithoutMaxFee } = transaction
+				return {
+					...transactionWithoutMaxFee,
+					...maxFeePerGas === 0n ? {} : { maxFeePerGas }
+				}
+			})
+			return this.executionSpec383MultiCallOnlyTransactionsAndSignatures(transactionsWithRemoveZeroPricedOnes, spoofedSignatures, blockNumber, extraAccountOverrides)
```

Comment:
At some point we should do a find and replace of `multicall` with `simulate` (with various casing adjustments as needed).